stable/kommander/templates/ingress.yaml

```diff
@@ -11,9 +11,7 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: traefik
     traefik.frontend.rule.type: PathPrefixStrip
```

Comment:
I think this should be configurable by values.yaml. For the konvoy ui page, this annotation should not be PathPrefixStrip.